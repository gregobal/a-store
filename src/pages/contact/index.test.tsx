import {render} from "@testing-library/react";
import {MemoryRouter} from "react-router-dom";
import {ContactPage} from "./index";

describe("Contact Page", () => {
    it("snapshot", () => {
        const view = render(
            <MemoryRouter>
                <ContactPage/>
            </MemoryRouter>
        );

        expect(view).toMatchSnapshot();
    });
});
