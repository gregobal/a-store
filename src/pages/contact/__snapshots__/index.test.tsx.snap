// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact Page snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <h1
        class="typography__component_h1nzy typography__styrene-xlarge_qohs3 typography__bold_h1nzy"
      >
        Контакты
      </h1>
      <div
        class="gap__gap_t9kgt gap__vertical_t9kgt"
        data-gap-size="4xl"
      />
      <span
        class="typography__primary-medium_xcdcz typography__bold_xcdcz"
      >
        <a
          class="link__component_1r33z link__primary_1b8mz link__withoutUnderline_1r33z"
          href="tel: +7 906 061 60 20"
        >
          <span
            class="link__text_1r33z"
          >
            +7 906 061 60 20
          </span>
        </a>
      </span>
      <span
        class="typography__primary-medium_xcdcz typography__bold_xcdcz"
      >
        <a
          class="link__component_1r33z link__primary_1b8mz link__withoutUnderline_1r33z"
          href="mailto: info@alfabankstore.ru"
        >
          <span
            class="link__text_1r33z"
          >
            info@alfabankstore.ru
          </span>
        </a>
      </span>
      <div
        class="gap__gap_t9kgt gap__vertical_t9kgt"
        data-gap-size="l"
      />
      <span
        class="typography__primary-medium_xcdcz typography__bold_xcdcz"
      >
        г. Москва, пр-т Андропова, 18 корп. 3
      </span>
      <div
        class="gap__gap_t9kgt gap__vertical_t9kgt"
        data-gap-size="l"
      />
      <span
        class="typography__primary-medium_xcdcz typography__bold_xcdcz"
      >
        пн-чт:
      </span>
      <span
        class="typography__primary-medium_xcdcz typography__bold_xcdcz"
      >
        10:00—19:00
      </span>
      <span
        class="typography__primary-medium_xcdcz typography__bold_xcdcz"
      >
        пт:
      </span>
      <span
        class="typography__primary-medium_xcdcz typography__bold_xcdcz"
      >
        10:00—17:30
      </span>
      <div
        class="gap__gap_t9kgt gap__vertical_t9kgt"
        data-gap-size="l"
      />
      <span
        class="typography__primary-medium_xcdcz typography__bold_xcdcz"
      >
        Принимаем к оплате карты Visa, Mastercard, МИР.
      </span>
      <div
        class="gap__gap_t9kgt gap__vertical_t9kgt"
        data-gap-size="xl"
      />
      <div
        class="skeleton__component_jp9h2 skeleton__animate_jp9h2"
      >
        <div
          class="wrapper"
        >
          <div
            class="container"
          />
        </div>
      </div>
      <div
        class="gap__gap_t9kgt gap__vertical_t9kgt"
        data-gap-size="xl"
      />
      <span
        class="typography__primary-small_xcdcz typography__bold_xcdcz"
      >
        <a
          class="link__component_1r33z link__primary_1b8mz"
          href="/policy"
        >
          <span
            class="link__text_1r33z"
          >
            Политика конфиденциальности и обработки персональных данных
          </span>
        </a>
      </span>
    </div>
  </body>,
  "container": <div>
    <h1
      class="typography__component_h1nzy typography__styrene-xlarge_qohs3 typography__bold_h1nzy"
    >
      Контакты
    </h1>
    <div
      class="gap__gap_t9kgt gap__vertical_t9kgt"
      data-gap-size="4xl"
    />
    <span
      class="typography__primary-medium_xcdcz typography__bold_xcdcz"
    >
      <a
        class="link__component_1r33z link__primary_1b8mz link__withoutUnderline_1r33z"
        href="tel: +7 906 061 60 20"
      >
        <span
          class="link__text_1r33z"
        >
          +7 906 061 60 20
        </span>
      </a>
    </span>
    <span
      class="typography__primary-medium_xcdcz typography__bold_xcdcz"
    >
      <a
        class="link__component_1r33z link__primary_1b8mz link__withoutUnderline_1r33z"
        href="mailto: info@alfabankstore.ru"
      >
        <span
          class="link__text_1r33z"
        >
          info@alfabankstore.ru
        </span>
      </a>
    </span>
    <div
      class="gap__gap_t9kgt gap__vertical_t9kgt"
      data-gap-size="l"
    />
    <span
      class="typography__primary-medium_xcdcz typography__bold_xcdcz"
    >
      г. Москва, пр-т Андропова, 18 корп. 3
    </span>
    <div
      class="gap__gap_t9kgt gap__vertical_t9kgt"
      data-gap-size="l"
    />
    <span
      class="typography__primary-medium_xcdcz typography__bold_xcdcz"
    >
      пн-чт:
    </span>
    <span
      class="typography__primary-medium_xcdcz typography__bold_xcdcz"
    >
      10:00—19:00
    </span>
    <span
      class="typography__primary-medium_xcdcz typography__bold_xcdcz"
    >
      пт:
    </span>
    <span
      class="typography__primary-medium_xcdcz typography__bold_xcdcz"
    >
      10:00—17:30
    </span>
    <div
      class="gap__gap_t9kgt gap__vertical_t9kgt"
      data-gap-size="l"
    />
    <span
      class="typography__primary-medium_xcdcz typography__bold_xcdcz"
    >
      Принимаем к оплате карты Visa, Mastercard, МИР.
    </span>
    <div
      class="gap__gap_t9kgt gap__vertical_t9kgt"
      data-gap-size="xl"
    />
    <div
      class="skeleton__component_jp9h2 skeleton__animate_jp9h2"
    >
      <div
        class="wrapper"
      >
        <div
          class="container"
        />
      </div>
    </div>
    <div
      class="gap__gap_t9kgt gap__vertical_t9kgt"
      data-gap-size="xl"
    />
    <span
      class="typography__primary-small_xcdcz typography__bold_xcdcz"
    >
      <a
        class="link__component_1r33z link__primary_1b8mz"
        href="/policy"
      >
        <span
          class="link__text_1r33z"
        >
          Политика конфиденциальности и обработки персональных данных
        </span>
      </a>
    </span>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
